cmd/clusterctl/internal/util/yaml.go

```diff
@@ -103,3 +106,27 @@ func FromUnstructured(objs []unstructured.Unstructured) ([]byte, error) {
 
 	return JoinYaml(ret...), nil
 }
+
+func ordinalize(n int) string {
```

Comment:
wanna move this to `util` package?

Reply:
+1

Reply:
I can move this to util package.

If we find other uses for the go-humanize library, I don't mind pulling it in. 
Usually, I tend to not pull in other dependencies unless absolutely necessary. Since this "ordinalize" functionality is being used only here, I prefer not pulling in the library. 
That said, again, if there are other places in the CAPI that could use the go-humanize, I don't mind pulling it in.

Reply:
@vincepri Just to clarify, you are suggesting to move this to `cluster-api/util` package. Correct?